test.js

```diff
@@ -1,5 +1,20 @@
 var t = "hello";
 
-if (t === "hello') {
-    console.log('ok');
+var selec = '[data-dikkemik]';
+var doofus = true;
+
+if (t == 'hello') {
+  console.log("ok");
+}
+
+if (true) {
+  var sjaakDeman = false;
+}
+
+if (true) {
+  var cat = false;
+}
+
+function that(){
+      return 'booyah'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 6.](https://www.codacy.com/app/rubstar/rvs-pr-testing/file/2480659477/issues/source?bid=3260976&fileBranchId=3261133#l19)